Jenkinsfile

```diff
@@ -39,7 +39,7 @@ pipeline {
                     def dockerImgVersion = ex_retrieveAppVersion()
                     // Build Docker image
                     sh """#!/bin/bash -xe
-                        docker build -f Dockerfile -t pdfy:${dockerImgVersion} .
+                        docker build -f Dockerfile -t linz-docker.jfrog.io/pdfy:${dockerImgVersion} .
                     """
```

Comment:
This change won't work. The tag here needs to match DOCKER_IMAGE_NAME from the infra file. Should stay as pdfy.

If you want to use Artifactory to avoid pulling from Dockerhub, adjust the Dockerfile instead.

Reply:
they don't have a dockerfile for this.. I reverse it then